x/feegrant/keeper/keeper_test.go

```diff
@@ -194,21 +212,28 @@ func (suite *KeeperTestSuite) TestUseGrantedFee() {
 	// some helpers
 	atom := sdk.NewCoins(sdk.NewInt64Coin("atom", 555))
 	eth := sdk.NewCoins(sdk.NewInt64Coin("eth", 123))
-	future := types.BasicFeeAllowance{
+	future := &types.FeeAllowance{Sum: &types.FeeAllowance_BasicFeeAllowance{BasicFeeAllowance: &types.BasicFeeAllowance{
 		SpendLimit: atom,
 		Expiration: types.ExpiresAtHeight(5678),
+	},
+	},
 	}
-	expired := types.BasicFeeAllowance{
+
+	expired := &types.FeeAllowance{Sum: &types.FeeAllowance_BasicFeeAllowance{BasicFeeAllowance: &types.BasicFeeAllowance{
 		SpendLimit: eth,
 		Expiration: types.ExpiresAtHeight(55),
+	},
+	},
 	}
 
 	// for testing limits of the contract
 	hugeAtom := sdk.NewCoins(sdk.NewInt64Coin("atom", 9999))
-	smallAtom := sdk.NewCoins(sdk.NewInt64Coin("atom", 1))
-	futureAfterSmall := types.BasicFeeAllowance{
-		SpendLimit: sdk.NewCoins(sdk.NewInt64Coin("atom", 554)),
+	smallAtom := sdk.NewCoins(sdk.NewInt64Coin("atom", 2))
```

Comment:
`1` looks small. Don't change it
```suggestion
	smallAtom := sdk.NewCoins(sdk.NewInt64Coin("atom", 1))
```